eng/tools/generate_api_report/README.md

```diff
@@ -7,13 +7,25 @@ It uses the following command `cargo +nightly rustdoc -Z unstable-options --outp
 
 ## Version Compatibility
 
-When updating the `rustdoc-types` in Cargo.toml, ensure that the `rust-api-parser` in the tools repository is also updated along with this project.
+The project depends on specific version relationships between several components:
 
-1. Check the `FORMAT_VERSION` value in the new version.
-2. Verify that it is compatible with your rustdoc JSON output version (generated by `cargo +nightly rustdoc ...`).
-3. If they do not match, you may need to use a different version of `rustdoc-types`.
+1. **NIGHTLY_TOOLCHAIN**: This value in main.rs must be set to a compatible nightly Rust version that produces the expected format version in the JSON output. Currently, the tool is pinned to a specific nightly (`nightly-2025-01-12`) to ensure consistent output.
```

Comment:
Except it isn't for dev environments. Only for the CI.

Reply:
works for everyone now :)